Riot/Modules/Common/CoachMessages/CoachMarkView.swift

```diff
@@ -0,0 +1,103 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import Reusable
+
+/// `CoachMarkView` is used to display an information bubble view with a given text.
+@objcMembers
+class CoachMarkView: UIView, NibLoadable, Themable {
+    
+    // MARK: Constants
+    
+    @objc enum MarkPosition: Int, RawRepresentable {
+        case topLeft
+        case zero
```

Comment:
How come this is an enum rather than passing in a point?

I'm wondering for example, if `.topLeft` is what is shown in the screenshot, then could there be times when a different origin is needed. Such as when the phone was in landscape or possibly dynamic type changing the side of the navigation bar?